config/webpack.config.js

```diff
@@ -96,6 +102,11 @@ var configGenerator = (options) => {
               NODE_ENV: JSON.stringify(process.env.NODE_ENV)
           }
       }),
+      new sassLintPlugin({
+        configFile: 'config/.sass-lint.yml',
```

Comment:
Don't use a dot-file here. Removing the leading dot keeps it easily visible in the filesystem.